sources/install/package_base.sh

```diff
@@ -345,6 +371,7 @@ function package_base_debug() {
     install_fzf                                         # Fuzzy finder
     install_pipx
     add-history curl
+    install_neovim
```

Comment:
Not sure this package is needed here, this debug section is only for docker image debug to work with lightweight images. Not exported as public images.